pkgs/tools/misc/mkbootimage/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libelf
+, pcre
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mkbootimage";
+  version = "2.3dev";
+
+  src = fetchFromGitHub {
+    owner = "antmicro";
+    repo = "zynq-mkbootimage";
+    rev = "872363ce32c249f8278cf107bc6d3bdeb38d849f";
+    hash = "sha256-5FPyAhUWZDwHbqmp9J2ZXTmjaXPz+dzrJMolaNwADHs=";
+  };
+
+  propagatedBuildInputs = [ libelf pcre ];
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```
We shouldn't override patchPhase. Please use postPatch instead.